src/usercloudssdk/client.py

```diff
@@ -469,6 +469,23 @@ def CreateToken(
         j = self._post("/tokenizer/tokens", data=ucjson.dumps(body))
         return j["data"]
 
+    def LookupOrCreateTokens(
+        self,
+        data: list[str],
+        transformers: list[ResourceID],
+        access_policies: list[ResourceID],
+    ) -> list[str]:
+        body = {
+            "data": data,
+            "transformer_rids": [t.__dict__ for t in transformers],
+            "access_policy_rids": [a.__dict__ for a in access_policies],
```

Comment:
I added https://github.com/userclouds/sdk-python/pull/4
once landed this code can be ` "access_policy_rids": [asdict(a) for a in access_policies],`
you will only need to add
`from dataclasses import asdict`